sdk/python/pvsite_datamodel/sqlmodels.py

```diff
@@ -147,15 +154,6 @@ class ForecastValueSQL(Base, CreatedMixin):
         index=True,
     )
 
-    # This is redundant from the Forecast table, but it means we can save a JOIN for a very frequent
```

Comment:
isnt this the `forecast value` table - i think we still need it in there

Reply:
Sorry I forgot to mention this in the description - I'll add a line in there.

Basically the `site_uuid` belongs to the `forecasts` table, *but* I thought that duplicating it in the `forecast_values` table would save a `JOIN`, for one of our most common queries: getting the values of the last forecast for a given site or sites.

However, I realized that this doesn't exactly work, because of the missing `timestamp_utc` I'm adding in this PR. We need to check in the `forecast` table for that `timestamp_utc`. So we might as well check the `site_uuid` in there too. It's two many columns (both `timestamp_utc` and `site_uuid`) to just duplicate them, if feels cleaner to just do a `JOIN` after all. Speed-wise I don't think it actually will be slower.

So this migration is 2 changes:
* Remove the `site_uuid` from the `forecast_values`, as it is already in the `forecast` table. (After reflexion, having it in both was a bad idea)
* Add the `timestamp_utc` column to the `forecast` table.

Reply:
Sorry I forgot to mention this in the description - I'll add a line in there.

Basically the `site_uuid` belongs to the `forecasts` table, *but* I thought that duplicating it in the `forecast_values` table would save a `JOIN`, for one of our most common queries: getting the values of the last forecast for a given site or sites.

However, I realized that this doesn't exactly work, because of the missing `timestamp_utc` I'm adding in this PR. We need to check in the `forecast` table for that `timestamp_utc`. So we might as well check the `site_uuid` in there too. It's two many columns (both `timestamp_utc` and `site_uuid`) to just duplicate them, if feels cleaner to just do a `JOIN` after all. Speed-wise I don't think it actually will be slower.

So this migration is 2 changes:
* Remove the `site_uuid` from the `forecast_values`, as it is already in the `forecast` table. (After reflexion, having it in both was a bad idea)
* Add the `timestamp_utc` column to the `forecast` table.